server/src/main/java/com/hedera/block/server/producer/Util.java

```diff
@@ -34,23 +32,14 @@ private Util() {}
      *
      * @param blockItem the block item to get the fake hash for
      * @return the fake hash for the given block item
-     * @throws IOException thrown if an I/O error occurs while getting the fake hash
      * @throws NoSuchAlgorithmException thrown if the SHA-384 algorithm is not available
      */
-    public static byte[] getFakeHash(BlockItem blockItem)
-            throws IOException, NoSuchAlgorithmException {
-
-        try (final ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
-                final ObjectOutputStream objectOutputStream =
-                        new ObjectOutputStream(byteArrayOutputStream)) {
-            objectOutputStream.writeObject(blockItem);
+    public static byte[] getFakeHash(BlockItem blockItem) throws NoSuchAlgorithmException {
```

Comment:
This isn't fake anymore...

Reply:
It's fake until there's a story to implement it. That's not part of this epic.